moveit_ros/moveit_servo/src/servo_calcs.cpp

```diff
@@ -787,11 +787,13 @@ bool ServoCalcs::enforcePositionLimits()
   {
     // Halt if we're past a joint margin and joint velocity is moving even farther past
     double joint_angle = 0;
+    std::size_t idx = 0;
     for (std::size_t c = 0; c < original_joint_state_.name.size(); ++c)
```

Comment:
I think you could use `idx` in the for loop and completely get rid of `c`.
```suggestion
    for (idx = 0; idx < original_joint_state_.name.size(); ++idx)
```

Reply:
That makes sense! i'll update that and test again

Reply:
yep! tested again and works